Source/WebCore/Modules/webxr/WebXRSession.cpp

```diff
@@ -509,6 +510,21 @@ void WebXRSession::applyPendingRenderState()
     }
 }
 
+void WebXRSession::minimalUpdateRendering()
+{
+    // Spec is unclear about this, but we have to execute any scheduled video frame updates for
+    // videos to work in WebXR if the page is backgrounded
+    if (!m_shouldServiceRequestVideoFrameCallbacks)
+        return;
+
+    auto& document = downcast<Document>(*scriptExecutionContext());
```

Comment:
This should be:
```
    RefPtr sessionDocument = downcast<Document>(scriptExecutionContext());
```

And the resulting pointer null checked, as is done elsewhere in this file.